src/parser.pegjs

```diff
@@ -48,3 +52,64 @@ digit
 
 hex_digit
 	= [0-9a-f]i
+
+// RSQL "quoted" values extesion
+// inspired by https://github.com/pegjs/pegjs/blob/master/examples/javascript.pegjs#L271
+
```

Comment:
I would like to have the added rules formatted the same way as the already existing ones: indented by tabulators instead of by two spaces.